src/xcode/ENA/ENA/Source/Scenes/Events/OnBehalfCheckinSubmission/TraceLocationSelection/OnBehalfTraceLocationSelectionViewModel.swift

```diff
@@ -19,6 +19,12 @@ class OnBehalfTraceLocationSelectionViewModel {
 		self.traceLocationCellModels = traceLocations
 			.map { TraceLocationSelectionCellModel(traceLocation: $0) }
 		self.cameraAuthorizationStatus = cameraAuthorizationStatus
+		
+		#if DEBUG
+		if isUITesting && LaunchArguments.traceLocation.emptyTraceLocations.boolValue == true {
+			self.traceLocationCellModels = []
+		}
+		#endif
```

Comment:
I think it would make sense to solve this at the store level, maybe use the `MockEventStore` for UI tests? Because with this solution the trace location overview and potentially other parts of the app would still have trace locations, so we would have an inconsistent app state in the tests.